gstreamer/build.gradle.kts

```diff
@@ -91,5 +67,6 @@ setupGenSources {
 }
 
 tasks.javadoc {
+    dependsOn(project(":glib").tasks.javadoc)
```

Comment:
Ideally, this dependency should be handled by linksOffline.
Maybe our existing extension method could be replaced by an extension method for Javadoc?
Could probably be implemented as follows:
```kt
fun Javadoc.linksOffline(url: String, project: Project) {
    if (options is StandardJavadocDocletOptions) {
        val task = project.tasks.named<Javadoc>("javadoc").get()
        (options as StandardJavadocDocletOptions).linksOffline(url, task.destinationDir.toString())
        dependsOn(task)
    } else throw TypeCastException("Unexpected javadoc options type")
}
```

Reply:
That worked perfectly. (commit ffcd150)